<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewUserWindow</class>
 <widget class="QWidget" name="NewUserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-size:18px;
color:rgb(208, 208, 208);
background:rgb(163, 96, 240);
}

QLabel
{
font-size: 30px;
}

#errorMessageLabel
{
background:transparent;
color: rgb(255, 0, 0);
font-size: 16px;
}

QFrame
{
background:#333;
border-radius: 15px;
}

QPushButton
{
border: 1px rgb(255, 141, 144);
background:rgb(255, 141, 144);
border-radius:9px;
color: #333;
}
QPushButton:hover:!pressed
{
background:rgb(204, 114, 117);
border-width:2px;
border-color: white;
border-style:solid;
border-radius:9px;
color: #333;
}

QLineEdit
{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;
}

#errorLineEdit
{
background:transparent;
border:none;
color:red;
}

QDateEdit
{
background: #333;
font-size: 16px;
color:#717072;
}

QGroupBox
{
background: transparent;
border:none;
color:#717072;
}

QComboBox
{
background: transparent;
border:none;
color:#717072;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>601</width>
     <height>201</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="firstNameEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>First name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lastNameEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Last name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="birthdayBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Birthday</string>
    </property>
    <widget class="QDateEdit" name="birthdayEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>163</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="showGroupSeparator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="displayFormat">
      <string>MM/dd/yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="genderComboBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Gender</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>160</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="errorLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="iconBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>641</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
 background-color: rgb(255, 141, 144);
 border-radius:40px;
 max-width:80px;
 max-height:80px;
 min-width:80px;
 min-height:80px;
}

QPushButton:hover:!pressed
{
 background-color: rgb(255, 141, 144);
 border-width:2px;
 border-radius:40px;
 border-color: white;
 border-style:solid;
 max-width:80px;
 max-height:80px;
 min-width:80px;
 min-height:80px;
}

QPushButton:checked
{
 background-color: rgb(255, 141, 144);
 border-width:2px;
 border-radius:40px;
 border-color: white;
 border-style:solid;
 max-width:80px;
 max-height:80px;
 min-width:80px;
 min-height:80px;
}
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="icecreamIconButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/images/images/ice-cream.png</normaloff>:/images/images/ice-cream.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">userIconGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="carIconButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/images/images/car.png</normaloff>:/images/images/car.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">userIconGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="dogIconButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/images/images/dog.png</normaloff>:/images/images/dog.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">userIconGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="moonIconButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/images/images/moon.png</normaloff>:/images/images/moon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">userIconGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="planeIconButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/images/images/plane.png</normaloff>:/images/images/plane.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">userIconGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="sunIconButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/images/images/sun.png</normaloff>:/images/images/sun.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">userIconGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="treeIconButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/images/images/tree.png</normaloff>:/images/images/tree.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">userIconGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="catIconButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/images/images/cat.png</normaloff>:/images/images/cat.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">userIconGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="monsterIconButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/images/images/monster.png</normaloff>:/images/images/monster.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">userIconGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="heartIconButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/images/images/heart.png</normaloff>:/images/images/heart.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">userIconGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="starIconButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/images/images/star.png</normaloff>:/images/images/star.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">userIconGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="wheatIconButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>100</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/images/images/wheat.png</normaloff>:/images/images/wheat.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>true</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">userIconGroup</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>430</y>
     <width>301</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="userIconGroup"/>
 </buttongroups>
</ui>
